src/dreyfus_index.erl

```diff
@@ -157,9 +157,29 @@ handle_call(info, _From, State) -> % obsolete
     Reply = info_int(State#state.index_pid),
     {reply, Reply, State}.
 
+
+handle_cast({ddoc_updated, DDocResult}, #state{} = State) ->
+    #index{sig = Sig} = State#state.index,
+    KeepIndex = case DDocResult of
```

Comment:
Should this be called KeepIndexes? If I understand correctly, if one index in the new design doc has the same signature, then we don't kill the existing indexes.  So if I had 1000 indexes, and I update 999 of them with a new definition (and thus a new signature), but only 1 still was unchanged, then that would mean 999 old indexes would still keep running right?

Reply:
@tonysun83 `KeepIndex` here means if we should keep the current `dreyfus_index` process or kill it.  It refers to a current single `dreyfus_index` process.  May be a better name could be `KeepIndexProcess`?

Reply:
@tonysun83 The intention is to shutdown all changed indexes, so in your case all 999 old indexes will be killed

Reply:
@mayya-sharipova : understand now, had to re-read it again, not I got it